sdk/storage/storage-file-share/test/fileserviceclient.spec.ts

```diff
@@ -512,7 +512,7 @@ describe("FileServiceClient Premium", () => {
     assert.ok(propertiesSet.protocol?.smb?.multichannel);
   });
 
-  it("Share Enable Protocol & Share Squash Root", async function (this: Context) {
+  it.skip("Share Enable Protocol & Share Squash Root", async function (this: Context) {
```

Comment:
This has been manually tested, so it's OK to skip temperately for this release. 
We should finally update test pipeline to create storage account which can run this test case and enable the case again.

Reply:
Will enable this case later.